spec/support/features/notice_helpers.rb

```diff
@@ -0,0 +1,5 @@
+module Features
+  def user_sees_error(text)
+    expect(page).to have_css '.flash-error', text: text
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.